### PR TITLE
Identifiers/UUID

### PR DESCRIPTION
**Pull request by sergiogragera**
UUID initial approach as we have discussed on #33

### CODE REVIEW

File: src/modules/commands/delete.command.ts [hunk 1]
```diff
@@ -1,15 +1,10 @@
-import {CustomDomainFactory} from "../domains/custom/custom.domain.factory";
-import {Domain} from "../domains/domain";
 import {Service} from "../services/services.crud";
 
 export const deleteCommand = async (request, reply) => {
-    const domainObjectsFactory = new CustomDomainFactory(request.params.entity, request.query);
-    const entity : Domain = domainObjectsFactory.instantiate();
     const crudService = request.diScope.resolve('crudService') as Service
-    crudService.delete(entity);
+    crudService.delete(request.params.id);
```
Comment (nmanero) [inline src/modules/commands/delete.command.ts:5]: I would keep sending the entity, then we forget about strings, ints...

Comment (sergiogragera) [reply src/modules/commands/delete.command.ts:5]: Good idea.

**Comment by sergiogragera** [comment]
I have updated local array repository to local dict with uuid as key and repository methods getById, add, update and delete have been refactored.

**Review by nmanero — APPROVED** [review]
Perfect!

**Pull request merged**